docs/sources/explore/logs-integration.md

```diff
@@ -16,81 +16,88 @@ weight: 15
+{{< /admonition >}}
 
 ### Escaping newlines
 
-Explore automatically detects some incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). When it detects such sequences, Explore provides an "Escape newlines" option.
+Explore automatically detects some incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). When it detects such sequences, Explore provides an **Escape newlines** option.
 
 To automatically fix incorrectly escaped sequences that Explore has detected:
 
-1. Click "Escape newlines" to replace the sequences.
-2. Manually review the replacements to confirm their correctness.
+1. Click **Escape newlines** to replace the sequences.
+2. Review returned log lines.
 
-Explore replaces these sequences. When it does so, the option will change from "Escape newlines" to "Remove escaping". Evaluate the changes as the parsing may not be accurate based on the input received. You can revert the replacements by clicking "Remove escaping".
+Explore replaces these sequences, changing the option from **Escape newlines** to **Remove escaping**. Assess the changes, as the parsing may not be accurate based on the input. To revert the replacements, click **Remove escaping**.
 
 ### Log level
 
-For the logs where a `level` label is specified, we use the value of this label to determine the log level and update color of each log line accordingly. If the log doesn't have specified level label, we try to find out if its content matches any of the supported expressions (see below for more information). The log level is always determined by the first match. In the case where Grafana is not able to infer a log level field, it will be visualized with an unknown log level.
+For logs where a `level` label is specified, the value of this label is used to determine the log level and update the color of each log line accordingly.
+If the log doesn't have a specified level label, Grafana attempts to determine if its content matches any of the supported expressions. See the table below for more information. The log level is always determined by the first match. If Grafana isn't able to infer a log level field, it gets visualized as an unknown log level.
 
-> **Tip:** If you use a Loki data source and the "level" is part of your log line, you can use parsers (JSON, logfmt, regex,..) to extract the level information into a level label that is used to determine the level value. This will allow the histogram to show the various log levels as separate bars.
+{{< admonition type="tip" >}}
+When using the Loki data source, if `level` is part of your log line, you can use parsers such as JSON, logfmt, or regex to extract the level information into a level label. This label is used to determine the level value, allowing the histogram to display the various log levels as separate bars.
```

Comment:
I think they should be in code because they are part of the query language: https://grafana.com/docs/loki/latest/send-data/promtail/stages/json/

```suggestion
When using the Loki data source, if `level` is part of your log line, you can use parsers such as `json`, `logfmt`, or `regex` to extract the level information into a level label. This label is used to determine the level value, allowing the histogram to display the various log levels as separate bars.
```